frontend/packages/console-shared/src/components/formik-fields/InputGroupField.tsx

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable no-unused-vars, no-undef */
+import * as React from 'react';
+import { InputGroup, TextInput, TextArea, TextInputTypes } from '@patternfly/react-core';
+import { GroupInputProps, BaseInputFieldProps, TextType } from './field-types';
+import BaseInputField from './BaseInputField';
+
+export type InputGroupFieldProps = GroupInputProps &
```

Comment:
Same as above. Why not just extend the type in field-types file. You already have a declaration there for `InputGroupField` in that file.